src/syscall_table/aarch64.rs

```diff
@@ -0,0 +1,312 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+// This file is auto-generated by `tools/generate_syscall_tables`.
+// Do NOT manually edit!
+// Generated on: Tue Sep 14 11:46:41 UTC 2021
+// Kernel version: 5.10
+
+use std::collections::HashMap;
+
+pub(crate) fn make_syscall_table() -> HashMap<&'static str, i64> {
+    vec![
```

Comment:
Nit: I think these are easier to read in syscall number order.

Reply:
They're sorted alphabetically. I don't have strong feelings about doing this either way, I think it depends on preference